foundations/05-descendant-combinator/solution/solution.css

```diff
@@ -3,4 +3,12 @@
   color: red;
   font-size: 20px;
   text-align: center;
-}
\ No newline at end of file
+}
+
+/* Other possible combinations of selectors
+
+div p
+div .text
+.container p
+
+*/
```

Comment:
Couple of things:

- Suggestion: changing line 8 to "Below are some other possible descendant combinators:"
- The file just needs an empty new line at the end of it

Reply:
My mind was stuck in the "imperative mood" due to git commits. Haha